### ISSUE TITLE
int-> size_t in admesh

### ISSUE THREAD

**Issue by supermerill** [slic3r/Slic3r#4633]
Hi
The admesh/stl.h has many int, can I replace them by size_t ?

I don't like the int keyword, it's definition vary from compiler to compiler, and it can be low (16b)

**Comment by lordofhyphens** [comment]
Yes, but it is probably worth submitting a PR to the admesh repo for that.

Be sure to remember to verify that anything replaced can't be negative.
